src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImpl.java

```diff
@@ -184,6 +189,18 @@ private boolean isIso20022FormatParameterEnabled(String parameterName) {
                 KFSConstants.Components.ISO_FORMAT, parameterName, Boolean.FALSE);
     }
 
+    private boolean shouldCreateLegacyCheckFiles() {
+        return true;
```

Comment:
Should this be driven off of a parameter rather than hard-coded?

Reply:
I agree that it makes sense to tie this to a non-hard-coded value. I'm wondering whether a parameter would be ideal for it, though, given that the setting will only have a short-term lifetime. (We won't need the legacy Check code once our local Check printing migrates to ISO 20022, and KualiCo retires/removes that legacy code in one of its 2023 releases). A config property would be a little easier to maintain but would not be as flexible as a parameter.

I'm trying to discern whether this should be set up as a config property, or if it should be set up as a parameter that will eventually be deleted in a future release. Please let me know if you have any thoughts on this.

Reply:
@cah292: May this will help to make the decision:
If we had to turn something off or on, which option would there be:

-  the least amount of disruption to the users
- require no system down time
- no required code rebuild or redeploy and could be done at a moment's notice

Thinking about it that way, I would lean toward a parameter that could be cleanup at a later date.

Reply:
@nkimble Thanks for the input. I was reluctant to create a parameter that I know we would be deleting in a few months, but upon further consideration, I'm fine with preparing a parameter-based approach instead. I will proceed with that option.

Reply:
I would lean toward a parameter

Reply:
I also think a parameter is the way to go here.